addons/binding/org.openhab.binding.astro/README.md

```diff
@@ -49,23 +50,57 @@ The things requires the geolocation (latitude, longitude) for which the calculat
     * **group** `position`
         * **channel** `azimuth, elevation` (Number)
 
+### Trigger Channels
+* **thing** `sun`
+    * **group** `rise, set, noon, night, morningNight, astroDawn, nauticDawn, civilDawn, astroDusk, nauticDusk, civilDusk, eveningNight, daylight`
+        * **event** `START, END`
+    * **group** `eclipse`
+        * **event**: `TOTAL, PARTIAL, RING`
+* **thing** `moon`
+    * **group** `rise`
+        * **event** `START`
+    * **group** `set`
+        * **event** `END`
+    * **group** `phase`
+        * **event**: `FIRST_QUARTER, THIRD_QUARTER, FULL, NEW`
+    * **group** `eclipse`
+        * **event**: `TOTAL, PARTIAL`
+    * **group** `perigee`
+        * **event**: `PERIGEE`
+    * **group** `apogee`
+        * **event**: `APOGEE`
+
+**Offsets:** For each event group you can optionally configure an offset in minutes.
+The offset must be configured in the channel properties for the corresponding thing.
+The minimum allowed offset is -1440 and the maximum allowed offset is 1440.
+
 ## Full Example
 
 Things:
 
 ```
-astro:sun:home  [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60]
-astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60]
+astro:sun:home  [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+```
+or optionally with an offset
+```
```

Comment:
(The markdown engine at docs.openhab.org is a bit more strict) You need to always leave one empty line before and after a code block. That's why an empty line is needed before and after this sentence.